src/test/java/testtype/CVSB_15130/IVATestUpdated_CVSB_15130_CAR.java

```diff
@@ -0,0 +1,456 @@
+package testtype.CVSB_15130;
+
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import pages.TestPage;
+import steps.*;
+import steps.composed.TestTypeCategoryComp;
+import steps.util.UtilSteps;
+import utils.BaseTestClass;
+
+@RunWith(SerenityRunner.class)
+public class IVATestUpdated_CVSB_15130_CAR extends BaseTestClass {
+
+    @Steps
+    UtilSteps utilSteps;
+
+    @Steps
+    PreparerSteps preparerSteps;
+
+    @Steps
+    TestSteps testSteps;
+
+    @Steps
+    TestPage testPage;
+
+    @Steps
+    TestTypeCategoryComp testTypeCategoryComp;
+
+    @Steps
+    TestTypeCategorySteps testTypeCategorySteps;
+
+    @Steps
+    TestTypeDetailsSteps testTypeDetailsSteps;
+
+    @Steps
+    EUVehicleCategorySteps euVehicleCategorySteps;
+
+    @Steps
+    OdometerReadingSteps odometerReadingSteps;
+
+    @Steps
+    TestTypeSubcategorySteps testTypeSubcategorySteps;
+
+    @Title("CVSB-15130 - AC1 - Certificate number for Failures - CAR - YA4, YB4")
+    @Test
+    public void testIVACertificateNumber_Failures_CAR_YA4() {
```

Comment:
Should the method name be amended now that more than one code is tested inside it?

Reply:
Dont think need to add all the codes in the method name